container_registry/container_analysis/src/sample/samples.go

```diff
@@ -173,13 +202,20 @@ func getDiscoveryInfo(ctx context.Context, client *containeranalysis.GrafeasV1Be
 	return nil
 }
 
-// [END discovery_info]
+// [END containeranalysis_discovery_info]
 
-// [START occurrences_for_note]
+// [START containeranalysis_occurrences_for_note]
 
 // getOccurrencesForNote retrieves all the Occurrences associated with a specified Note.
 // Here, all Occurrences are printed and counted.
```

Comment:
Same printing comment (probably more throughout).

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#print-to-an-iowriter